core/jobs/transforms/job_result_transforms.py

```
@@ -22,9 +22,86 @@
 
 import apache_beam as beam
 import result
+from apache_beam import pvalue
 from typing import Any, Optional, Tuple
```

Ruff (0.15.15)

[warning] `typing.Tuple` is deprecated, use `tuple` instead

(UP035)